ScoutSuite/providers/aws/provider.py

```diff
@@ -78,7 +78,7 @@ def preprocessing(self, ip_ranges=None, ip_ranges_name_key=None):
 
         self._add_cidr_display_name(ip_ranges, ip_ranges_name_key)
         self._merge_route53_and_route53domains()
-        self._match_iam_policies_and_buckets()
+        # self._match_iam_policies_and_buckets() # TODO: Uncomment this!
```

Comment:
:thinking_face: j/k this can be done afterwards :slightly_smiling_face:

Reply:
Woopsi, good think I put a todo hehe! Thanks!